.gitlab-ci.yml

```diff
@@ -333,6 +365,8 @@ build_qemux86_64_uefi_grub:
   after_script:
     - docker save mendersoftware/mender-client-qemu:pr -o mender-client-qemu_qemux86_64_uefi_grub.tar
     - docker save mendersoftware/mender-client-qemu-rofs:pr -o mender-client-qemu-rofs_qemux86_64_uefi_grub.tar
+    # Post job status
+    - ${CI_PROJECT_DIR}/scripts/github_pull_request_status $(cat /JOB_RESULT.txt) "${CI_JOB_NAME} started" "${CI_JOB_URL}" "Gitlab_${CI_JOB_NAME}"
```

Comment:
Here an in almost all the following, the message should be `"${CI_JOB_NAME} finished"`. Same logic as in the init job: started + finished

Sorry I have missed this in the first review.

Reply:
Ah, of course. Good catch.